extensions/GlobalMessages/GlobalMessagesG.i18n.php

```diff
@@ -52,9 +52,7 @@
 	'group-bot.js' => '/* Any JavaScript here will be loaded for bots only */',
 	'group-sysop.js' => '/* Any JavaScript here will be loaded for sysops only */',
 	'group-bureaucrat.js' => '/* Any JavaScript here will be loaded for bureaucrats only */',
-	'group-vstf' => 'VSTF',
-	'group-vstf-member' => 'VSTF',
-	'grouppage-vstf' => 'w:Wikia:SpamTaskForce',
+	'grouppage-soap' => 'w:Wikia:SpamTaskForce',
```

Comment:
This apply to all of this "links" should they also be changed to `SOAP` or we don't have this pages/redirect in place yet?

Reply:
I've assumed that the page will be the same but content will change, I'll ask ticket creator